cypress/integration/Util.ts

```diff
@@ -57,3 +57,35 @@ export function deletedInstanceNotExist(instanceName: string) {
     );
   });
 }
+
+/**
+ * Assert if the test environment is set by the expected value.
+ * @param type a name of the desired test environment
+ * @returns
+ */
+export function isEnvironmentType(type: string) {
```

Comment:
can't we use `type: EnvType` instead of `type: string`?

Reply:
fixed